app/ethermint.go

```diff
@@ -144,7 +147,8 @@ func NewEthermintApp(logger tmlog.Logger, db dbm.DB, loadLatest bool,
 		keyGov:         sdk.NewKVStoreKey(gov.StoreKey),
 		keyParams:      sdk.NewKVStoreKey(params.StoreKey),
 		tkeyParams:     sdk.NewTransientStoreKey(params.TStoreKey),
-		// TODO: Initialize evm module key
+		evmStoreKey: sdk.NewKVStoreKey(params.StoreKey),
+		evmCodeKey: sdk.NewKVStoreKey(params.StoreKey),
```

Comment:
These key values for where the KV store is located is duplicated, we need to create the defaults through our module and use these or else it will error out. The only time these key values are existing in the codebase is in the `importer_test.go`:

```
	storageKey = sdk.NewKVStoreKey("storage")
	codeKey    = sdk.NewKVStoreKey("code")
```

So we should make a public default inside the module with a more evm specific name (maybe "evm_storage" and "evm_code") for each to be used by this test but also in these lines so that the location in memory is correct

Reply:
Added keys to `types/key.go`, now used in import test and in app